docs/resources/api-key.md

```diff
@@ -66,7 +66,8 @@ resource "codefresh_team" "team_1" {
 ## Argument Reference
 
 - `name` - (Required) The display name for the API key.
-- `account_id` - (Required) The ID of account than should own the new API key.
+- `account_id` - (Required) The ID of account in which the API key will be created.
+- `user_id` - (Required) The ID of a user within the above account that will own the API key. 
```

Comment:
would it make sense to replace "the above account" with "the `account_id`"?
@YakShavingCatHerder

Reply:
@sharon-codefresh, agreed. Referencing the required var would be more apparent.